client/src/components/ButtonLogout/ButtonLogout.jsx

```diff
@@ -0,0 +1,39 @@
+import "./ButtonLogout.css";
+import axios from "axios";
+import PropTypes from "prop-types";
+
+export default function ButtonLogout({ setCurrentUser }) {
+  const handleLogout = async () => {
+    try {
+      await axios.get("http://localhost:3310/api/auth/logout", {
```

Comment:
replace this first part of your url with the environment variable API_URL

Reply:
Done